docs/How-To/cli/README.md

```diff
@@ -422,83 +245,65 @@ fundraisingd tx fundraising bid 1 1.0 5000000denom2 \
 --output json | jq
 ```
 
-Result:
+### ModifyBid
+This is for a bidder to modify an existing bid that the bidder placed previously. Note that this `ModifyBid` is supported only for `BatchAuction`. The bidder can modify the bid only with the same bid type, and also only with either higher bid price or larger bid amount.  
 
-```json
-{
-  "@type": "/cosmos.tx.v1beta1.Tx",
-  "body": {
-    "messages": [
-      {
-        "@type": "/tendermint.fundraising.MsgPlaceBid",
-        "auction_id": "1",
-        "bidder": "cosmos1tfzynkllgxdpmrcknx2j5d0hj9zd82tceyfa5n",
-        "price": "1.000000000000000000",
-        "coin": {
-          "denom": "denom2",
-          "amount": "5000000"
-        }
-      }
-    ],
-    "memo": "",
-    "timeout_height": "0",
-    "extension_options": [],
-    "non_critical_extension_options": []
-  },
-  "auth_info": {
-    "signer_infos": [
-      {
-        "public_key": {
-          "@type": "/cosmos.crypto.secp256k1.PubKey",
-          "key": "A8VLxM/RDIlFEtOe7rfzA2Am55/Zam2n+oq1+I/Ovkbv"
-        },
-        "mode_info": {
-          "single": {
-            "mode": "SIGN_MODE_DIRECT"
-          }
-        },
-        "sequence": "1"
-      }
-    ],
-    "fee": {
-      "amount": [],
-      "gas_limit": "200000",
-      "payer": "",
-      "granter": ""
-    }
-  },
-  "signatures": [
-    "Ahrvo4CXneHxTd0Hgyt+HdZXmrhKhm1ijo5Tf7/K7OcK4P5590UlDpoqJ7ofLB738AGt+3rJ+cHy+K09KqBFaA=="
-  ]
-}
+Usage
+```bash
+modify-bid [auction-id] [bid-id] [price] [coin]
+```
+
+| **Argument**      |  **Description**                     |
+| :---------------- | :----------------------------------- |
+| auction-id        | auction ID that the bid corresponds to. | 
+| bid-id    | bid ID that the bidder placed previously in this auction |
```

Comment:
I don't think we need to capitalize id imo (description).

Reply:
Checked, and agree to modify with small letters.